### ISSUE TITLE
Delete Event

### ISSUE THREAD

**Issue by Lucas-Martin2023** [Lucas-Martin2023/NeverForgetADate#9]
# User Story
As a user, I want the ability to delete an event that is no longer relevant or meaningful.

# Acceptance Criteria
- [ ] Implement a "Delete" button on the event details page.
- [ ] Display a confirmation prompt before proceeding with the deletion.
- [ ] Ensure the event is successfully deleted upon confirmation.

# EXAMPLE

# Dependencies
The Event cards will need to be completed first so the user has something to delete.

# Dev Notes
Create a reusable React component for the "Delete" button to trigger the deletion process.
Utilize React state to manage the confirmation prompt and user interaction.
Implement an API call to delete the event on confirmation.
Ensure proper error handling and user feedback during the deletion process.
